specification/web/resource-manager/Microsoft.Web/stable/2016-03-01/ResourceHealthMetadata.json

```diff
@@ -0,0 +1,373 @@
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ResourceHealthMetadataCollection"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List ResourceHealthMetadata for a site": {
+            "$ref": "./examples/ListResourceHealthMetadataBySite.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots/{slot}/resourceHealthMetadata/default": {
+      "get": {
+        "tags": [
+          "ResourceHealthMetadata"
+        ],
+        "summary": "Gets the category of ResourceHealthMetadata to use for the given site",
+        "description": "Gets the category of ResourceHealthMetadata to use for the given site",
+        "operationId": "ResourceHealthMetadata_GetBySiteSlot",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of web app",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "slot",
+            "in": "path",
+            "description": "Name of web app slot. If not specified then will default to production slot.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ResourceHealthMetadataCollection"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get ResourceHealthMetadata": {
+            "$ref": "./examples/GetResourceHealthMetadataBySite.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ResourceHealthMetadata": {
+      "description": "Used for getting ResourceHealthCheck settings.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "./CommonDefinitions.json#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "ResourceHealthMetadata resource specific properties",
+          "properties": {
+            "category": {
+              "description": "The category that the resource matches in the RHC Policy File",
+              "type": "string"
+            },
+            "signalAvailability": {
```

Comment:
ideally this should be an enum with multiple states. Currently you might just require true or false for this but in future as scenario evolves you might want to indicate more states.

Reply:
Same as category, this is a boolean as per the spec I was given by RHC team:
SignalAvailability (boolean) - This field is to notify us if the signal for a resource is reaching us or not. This can come into picture when you are aware that a particular set of resources (possibly belonging a test region or a legacy region) are not sending RHC signals. When this is false, we will put the corresponding resource into unknown events bucket.